pages.ml/common/cargo.md

```diff
@@ -0,0 +1,37 @@
+# cargo
+
+> റസ്റ്റ് പ്രോജക്റ്റുകളും അവയുടെ മൊഡ്യൂൾ ഡിപൻഡൻസികളും (ക്രാറ്റുകൾ) കൈകാര്യം ചെയ്യുക.
+> `ബിൽഡ്` പോലുള്ള ചില ഉപകമാൻഡുകൾക്ക് അവരുടേതായ ഉപയോഗ ഡോക്യുമെന്റേഷൻ ഉണ്ട്.
+> കൂടുതൽ വിവരങ്ങൾ: <https://doc.rust-lang.org/cargo>.
+
+- ക്രേയ്റ്റ്സ് തിരയുക:
+
+`cargo search {{ക്രേയ്‌റ്റ്സിന്റെ പേര്}}`
+
+- ഒരു ബൈനറി ക്രേയ്റ്റ് ഇൻസ്റ്റാൾ ചെയ്യുക:
+
+`cargo install {{ക്രേയ്‌റ്റ്സിന്റെ പേര്}}`
+
+- ഇൻസ്റ്റാൾ ചെയ്ത ബൈനറി ക്രേറ്റുകളുടെ പട്ടിക:
+
+`cargo install --list`
+
+- നിർദ്ദിഷ്ട ഡയറക്‌ടറിയിൽ ഒരു പുതിയ ബൈനറി അല്ലെങ്കിൽ ലൈബ്രറി റസ്റ്റ് പ്രോജക്‌റ്റ് സൃഷ്‌ടിക്കുക (അല്ലെങ്കിൽ സ്ഥിരസ്ഥിതിയായി നിലവിലുള്ള ഡയറക്ടറിയിൽ):
+
+`cargo init --{{bin|lib}} {{ഫയലിലേക്കുള്ള/പാത}}`
+
+- നിലവിലെ ഡയറക്‌ടറിയിൽ `Cargo.toml`-ലേക്ക് ഒരു ഡിപൻഡൻസി ചേർക്കുക:
+
+`cargo add {{ഡിപൻഡൻസിയുടെ പേര്}}`
+
+- റിലീസ് പ്രൊഫൈൽ ഉപയോഗിച്ച് നിലവിലെ ഡയറക്ടറിയിൽ റസ്റ്റ് പ്രോജക്റ്റ് നിർമ്മിക്കുക:
+
+`cargo build --release`
+
+- നൈറ്റ്ലി കംപൈലർ ഉപയോഗിച്ച് നിലവിലെ ഡയറക്‌ടറിയിൽ റസ്റ്റ് പ്രോജക്റ്റ് നിർമ്മിക്കുക (`rustup` ആവശ്യമാണ്):
+
+`cargo +nightly build`
+
+- ഒരു നിശ്ചിത എണ്ണം ത്രെഡുകൾ ഉപയോഗിച്ച് നിർമ്മിക്കുക (സ്ഥിരസ്ഥിതി ലോജിക്കൽ സിപിയുകളുടെ എണ്ണമാണ്):
+
+`cargo build --jobs {{number_of_threads}}`
```

Comment:
If possible, can you translate this too (if there isn't an equivalent word feel free to skip this suggestion)?